src/Keboola/DbExtractor/Application.php

```diff
@@ -8,6 +8,7 @@
 use Keboola\DbExtractor\Configuration\ConfigDefinition;
 use Keboola\DbExtractor\Configuration\ConfigRowDefinition;
 use Keboola\DbExtractor\Exception\UserException;
+use Keboola\DbExtractor\Logger\Logger;
```

Comment:
IMHO by to nemělo mít společný namespace. Ale `Keboola\DbExtractorLogger`...

Reply:
přejmenoval jsem to

Reply:
Ideálně ale všechno (SSH, RetryProxy, ...)

Reply:
jj je to vše ;)